missing library for native-image on "findepi/graalvm:20.3.0-java11-native"

trying to build a native image using `findepi/graalvm:20.3.0-java11-native` but i get the following error.

```
/usr/bin/ld: cannot find -lstdc++
collect2: error: ld returned 1 exit status
```
it looks like the standard c++ library is missing.

please add this to the base image.
```
 apt-get install -y g++
```

--- a/Dockerfile.native
+++ b/Dockerfile.native
@@ -8,6 +8,7 @@ RUN set -xeu && \
     apt-get update && \
     apt-get install -y \
         gcc \
+        g++ \
         libz-dev \
         && \
     gu install native-image && \

